src/compute_embedding.py

```diff
@@ -0,0 +1,44 @@
+import pandas as pd
+import numpy as np
+import tiktoken
+import os
+import openai
+from openai.embeddings_utils import get_embedding
+from tokenizer import get_token_from_string 
+
+def embedding_from_string(string: str, embedding_name: str, encoding_name: str, max_token: int = 8191) -> list[float]:
+    """
+    computes embedding from a string
+    input: string
+    output: embedding for string (1536 dimensional float vector)
+    """
+    # check if number of tokens is too large to compute embedding
+    if(max_token > 8191):
```

Comment:
```suggestion
    if (max_token > 8191):
```
Like here: https://peps.python.org/pep-0008/